### ISSUE TITLE
[CST] Understand how claim letter PDFs are made

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#96066]
## Value Statement

**_As a_** Veteran
**_I want to_** access accessible claim letter PDFs
**_So that_** I can read them and store them how I want to on my own device

---
### New Feature

We suspect that claim letter PDFs aren't accessible (work happening on this in #95017), but we don't exactly what's wrong with the PDFs. In order to potentially deliver accessible PDFs, we need to understand what's making them inaccessible, and **how they're created, if inaccessible**. According to @jstrothman - "While our team doesn't generate the letters, we do deliver them, so I think we should at the very least know what, if any, accessibility issues exist." For more, [check out this Slack thread](https://dsva.slack.com/archives/C06V3RU6HA9/p1725993257419269).

## Outcome, Success Measure, KPI(S), and Tracking Link
- Add here

## Design

From an accessibility perspective, we will:

- Talk to the team/stakeholders that create the PDFs to understand that process and would could prevent the PDFs from being made accessible.
- From there, we can outline some possibilities around how to make the PDFs accessible so that the enablement team and the product team can potentially prioritize work.

List of letters:

- 184 (doc type) - Decision Notices
- 34 - Correspondence
- 408 - VA Examination Letter
- 700 - MAP-D Development Letter
- 859 - Subsequent Development Letter
- 940 - Final Attempt Letter
- 864 - General Records Request (Medical)
- 1605 - General Records Request (Non-Medical)
- 704 - 5103
- 706 - 5103
- 858 - 5103

### Enablement team (if needed)

@jstrothman, for awareness.

## Engineering 

N/A.

## Out of scope

- A lot of the possible fixes may be out of our team's control, and would need to be shared with other teams.

## Open questions

- Here, I want to outline what teams control the PDFs, so I can talk to them, if necessary.

## Tasks
- [x] Determine how PDFs get created.

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [x] Documentation created for how PDFs get created and how it impacts accessibility.

**Comment by davidakennedy** [comment]
Here's an update on this work so far:

- I've started a draft of all the information related to the accessibility review of PDFs and what I've learned through the work on this ticket. It's in internal review and I'll share with Julie soon.
- So far, what I know is: Some PDFs are made by an Adobe tool/service called the Documentation Generation API. Other PDFs are made by an internal tool to the VA called the Letter Creator. The Letter Creator runs off of an Excel spreadsheet. The Letter Creator's are more accessible than the ones made by the Documentation Gerneration API because they have "tags." When a PDF is "tagged," it has the role needed behind the text to provide extra meaning to assistive technology.
- We're having a meeting today with Julie, Benefits stakeholders and accessibility subject matter experts to talk about next steps.

**Comment by davidakennedy** [comment]
Following up here to walk through the artifacts for this completed work:

- All the work for this ticket is captured in the "claim letters accessibility" folder in Sharepoint.
- In that folder, is another called, "claim-letters-tested-for-accessibility". This includes the test PDFs that were tested.
- The main document with all of the test findings and recommendations is called "2024-12 Claim letter pdfs accessibility findings.docx".
- The main folder also includes a mp4 video file with a demo of a screen reader experience for one of the PDFs.

This work is complete now. cc: @jacobworrell
